main/inc/lib/MoodleImport.lib.php

```diff
@@ -0,0 +1,733 @@
+<?php
```

Comment:
This file should be called MoodleImport.class.php. The .lib.php is an old extension and we're trying to move away from it.